pkg/objectupdate/rte/rte_test.go

```diff
@@ -349,3 +349,63 @@ func TestSecurityContext(t *testing.T) {
 		})
 	}
 }
+
+func TestSecurityContextWithOpts(t *testing.T) {
+	testCases := []struct {
+		description        string
+		selinuxContextType string
+		sccName            string
+	}{
```

Comment:
would have been better to have a reference object to compare equality for in the test body, but also this approach is acceptable.